### ISSUE TITLE
How to pass `--shm-size="2g"` to docker run?

### ISSUE THREAD

**Issue by mwinkler** [dotnet/tye#1288]
Hi

Along with my application I want to start the docker image `selenium/node-chrome`.
Regarding its documentation (-> https://github.com/SeleniumHQ/docker-selenium), 
the argument `--shm-size="2g"` should be passed to `docker run`.
Currently I see no option in `tye.yaml` to achieve this.

I can start the selenium chrome container without this option, but the browser inside the container crashes.
If I start the container by myself with the `--shm-size="2g"` argument, it works without any problems.

Thanks for help.

**Comment by philliphoff** [comment]
@mwinkler You're right in that it doesn't appear that arbitrary arguments can be passed to the `docker run` command, nor does `tye.yaml` specifically understand the `shm-size` setting.  That would be a useful thing to have (e.g. a `dockerRunArgs` property that parallels the existing `dockerFileArgs` property used to pass arbitrary arguments to `docker build`.

Unfortunately, I don't see an immediate workaround for this specific setting.  You may be able to startup the service separately via `docker` command line, then refer to it in the `tye.yaml` as an "external" service so it will be reachable from the other services.  You might also try defining the service as an "executable" service, using the full `docker` command as the executable/args.
